docs/release-checklist.rst

```diff
@@ -95,23 +113,28 @@ they will need to evaluate which contributors' signatures they trust.
 - (all steps above are completed)
 - sign the release
 
-  - git tag -s -u 0xE34E62D06D0E69CFCA4179FFBDE0D31D68666A7A -m "release Tahoe-LAFS-1.15.0rc0" tahoe-lafs-1.15.0rc0
-  - (replace the key-id above with your own)
+  - ``git tag -s -u 0xE34E62D06D0E69CFCA4179FFBDE0D31D68666A7A -m "release Tahoe-LAFS-1.16.0rc0" tahoe-lafs-1.16.0rc0``
+
+.. note:: 
+    - Replace the key-id above with your own, which can simply be your email if it's attached to your fingerprint.
+    - Don't forget to put the correct tag message and name. In this example, the tag message is "release Tahoe-LAFS-1.16.0rc0" and the tag name is ``tahoe-lafs-1.16.0rc0`` 
 
 - build all code locally
+
   - these should all pass:
 
-    - tox -e py27,codechecks,docs,integration
+    - ``tox -e py27,codechecks,docs,integration``
```

Comment:
This actually leaves a bunch of garbage in the checkout, making the "clean checkout" part not really work .. specifically, building the docs means a bunch of that gets included in the final release (I believe .. might want to double-check).

Reply:
umm, so take docs out?

Reply:
No. We want to test that the docs build properly.
I think the "clean checkout" step is maybe in the wrong place -- we need instead is to ensure that any leftover cruft is gone immediately before the packaging step (see also Greg Troxel's recent mailing-list post following-up to 1.17.0 release).

I suspect that the other builds in there _could_ also introduce local cruft (if they don't already, they could in future).